code/src/components/NewPost.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+const NewPost = ({ handleFormSubmit, onNewPostChange, newPost }) => {
+  return (
+    <div className="newPost-wrapper">
+      <form onSubmit={handleFormSubmit}>
+        <label className="newPost-label" htmlFor="newPost">What is making you happy right now?
+          <textarea
+            className="newPost-input"
+            id="newPost"
+            type="text"
+            value={newPost}
+            maxLength="140"
+            onChange={onNewPostChange}
+            rows="3"
+            cols="40" />
+          <p style={{ color: newPost.length > 130 ? 'red' : 'black' }}>{newPost.length}/140</p>
+          <button type="submit"><span>❤️️</span> Send Happy Thought <span>❤️️</span></button>
```

Comment:
This button is a little bit small, would look great if it was just a bit wider because the second heart is on it's own line